File: swift/Sources/Schemas/TraceStreamRequestFormat.swift
import Foundation

public enum TraceStreamRequestFormat: String, Codable, Hashable, CaseIterable, Sendable {
    case openinferenceV1 = "openinference.v1"
    case opentelemetryOtlpV1 = "opentelemetry.otlp.v1"
    case harborAtifV1 = "harbor.atif.v1"
}

File: ruby/lib/Sikaru/trace_streams/types/trace_stream_request_format.rb
# frozen_string_literal: true

module Sikaru
  module TraceStreams
    module Types
      module TraceStreamRequestFormat
        extend Sikaru::Internal::Types::Enum

        OPENINFERENCE_V1 = "openinference.v1"
        OPENTELEMETRY_OTLP_V1 = "opentelemetry.otlp.v1"
        HARBOR_ATIF_V1 = "harbor.atif.v1"
      end
    end
  end
end
